yb-voyager/src/tgtdb/event.go

```diff
@@ -282,7 +282,7 @@ func (event *Event) getPreparedInsertStmt(tdb TargetDB, targetDBType string) (st
 	columns := strings.Join(columnList, ", ")
 	values := strings.Join(valueList, ", ")
 	stmt := fmt.Sprintf(insertTemplate, event.TableNameTup.ForUserQuery(), columns, values)
-	if targetDBType == POSTGRESQL {
+	if targetDBType == POSTGRESQL || targetDBType == YUGABYTEDB {
```

Comment:
Either we should use add the flag check here as well or remove it from everywhere and use upsert behavior directly without the check for events.